### DISCUSSION TITLE
Enable/Disable CodeQL code scanning for a repo using Github Rest API

### DISCUSSION THREAD

**Discussion by shrikantarchit** [github/codeql#12161] [Q&A]
Is it possible to enable/disable CodeQL code scanning for a repo using Github Rest API in any of the Github accounts?(Personal accounts/ Organization accounts /Enterprise accounts)

**Comment by alexet** [comment]
Hi

It is not possible at the moment to enable and disable default setup code scanning by rest api. If you enabled it via a workflow you could use the rest API to create and delete workflow files.
